src/index.js

```diff
@@ -80,7 +80,7 @@ function ParseServer({
   cloud,
   collectionPrefix = '',
   clientKey = '',
-  javascriptKey = randomString(20),
+  javascriptKey = '',
```

Comment:
because if the server believes it has any client-type key set, every request to the server requires a valid client key (js, client, dotnet, rest).